### ISSUE TITLE
Fix warnings for mingw builds

### ISSUE THREAD

**Issue by qwepoizt** [ElektraInitiative/libelektra#4072]
Currently, the mingw builds (see `scripts/dev/configure-mingw-w64-i686` and `scripts/dev/configure-mingw-w64-x86_64`) raise warnings.

This will be reproducible once #4070 is merged.

## Steps to Reproduce the Problem

1. `mkdir build && cd build`
2. ` ../scripts/dev/configure-mingw-w64-i686 ..`  or ` ../scripts/dev/configure-mingw-w64-x86_64 ..`
3. `make`

## Expected Result

Make succeeds without warnings.

## Actual Result

Make reports warnings.

## System Information

- Elektra Version: master

**Comment by markus2330** [comment]
Thank you for creating this issue!

> This will be reproducible once #4070 is merged.

Can you paste a link so that someone can find these warnings?

**Comment by qwepoizt** [comment]
The warnings can be seen in this log: https://build.libelektra.org/blue/rest/organizations/jenkins/pipelines/libelektra/branches/master/runs/577/nodes/940/log/?start=0

If I compile locally on Ubuntu 20.04, I receive additional warnings.

## Installed packages
```
$ dpkg -l | grep -E "(mingw|gcc|g\+\+|cmake)"|sort
ii  binutils-mingw-w64-i686                                     2.34-5ubuntu1+8.8                     amd64        Cross-binutils for Win32 (x86) using MinGW-w64
ii  binutils-mingw-w64-x86-64                                   2.34-5ubuntu1+8.8                     amd64        Cross-binutils for Win64 (x64) using MinGW-w64
ii  cmake                                                       3.16.3-1ubuntu1                       amd64        cross-platform, open-source make system
ii  cmake-data                                                  3.16.3-1ubuntu1                       all          CMake data files (modules, templates and documentation)
ii  cmake-extras                                                1.5-1                                 all          Extra CMake utility modules
ii  g++                                                         4:9.3.0-1ubuntu2                      amd64        GNU C++ compiler
ii  g++-9                                                       9.3.0-17ubuntu1~20.04                 amd64        GNU C++ compiler
ii  gcc-10-base:amd64                                           10.3.0-1ubuntu1~20.04                 amd64        GCC, the GNU Compiler Collection (base package)
ii  gcc-10-base:i386                                            10.3.0-1ubuntu1~20.04                 i386         GCC, the GNU Compiler Collection (base package)
ii  gcc                                                         4:9.3.0-1ubuntu2                      amd64        GNU C compiler
ii  gcc-9                                                       9.3.0-17ubuntu1~20.04                 amd64        GNU C compiler
ii  gcc-9-base:amd64                                            9.3.0-17ubuntu1~20.04                 amd64        GCC, the GNU Compiler Collection (base package)
ii  gcc-mingw-w64                                               9.3.0-7ubuntu1+22~exp1ubuntu4         all          GNU C compiler for MinGW-w64
ii  gcc-mingw-w64-base                                          9.3.0-7ubuntu1+22~exp1ubuntu4         amd64        GNU Compiler Collection for MinGW-w64 (base package)
ii  gcc-mingw-w64-i686                                          9.3.0-7ubuntu1+22~exp1ubuntu4         amd64        GNU C compiler for MinGW-w64 targeting Win32
ii  gcc-mingw-w64-x86-64                                        9.3.0-7ubuntu1+22~exp1ubuntu4         amd64        GNU C compiler for MinGW-w64 targeting Win64
ii  gdb-mingw-w64                                               9.1-0ubuntu1+10.9build1               amd64        Cross-debugger for Win32 and Win64 using MinGW-w64
ii  gdb-mingw-w64-target                                        9.1-0ubuntu1+10.9build1               all          Cross-debugger server for Win32 and Win64 using MinGW-w64
ii  g++-mingw-w64                                               9.3.0-7ubuntu1+22~exp1ubuntu4         all          GNU C++ compiler for MinGW-w64
ii  g++-mingw-w64-i686                                          9.3.0-7ubuntu1+22~exp1ubuntu4         amd64        GNU C++ compiler for MinGW-w64 targeting Win32
ii  g++-mingw-w64-x86-64                                        9.3.0-7ubuntu1+22~exp1ubuntu4         amd64        GNU C++ compiler for MinGW-w64 targeting Win64
ii  lib32gcc-s1                                                 10.3.0-1ubuntu1~20.04                 amd64        GCC support library (32 bit Version)
ii  libgcc-9-dev:amd64                                          9.3.0-17ubuntu1~20.04                 amd64        GCC support library (development files)
ii  libgcc-s1:amd64                                             10.3.0-1ubuntu1~20.04                 amd64        GCC support library
ii  libgcc-s1:i386                                              10.3.0-1ubuntu1~20.04                 i386         GCC support library
ii  mingw-w64                                                   7.0.0-2                               all          Development environment targeting 32- and 64-bit Windows
ii  mingw-w64-common                                            7.0.0-2                               all          Common files for Mingw-w64
ii  mingw-w64-i686-dev                                          7.0.0-2                               all          Development files for MinGW-w64 targeting Win32
ii  mingw-w64-x86-64-dev                                        7.0.0-2                               all          Development files for MinGW-w64 targeting Win64
```

## Log from local run 
```
$ ../scripts/dev/configure-mingw-w64-x86_64 ..

-- The C compiler identification is GNU 9.3.0
-- The CXX compiler identification is GNU 9.3.0
-- Check for working C compiler: /usr/bin/x86_64-w64-mingw32-gcc-posix
-- Check for working C compiler: /usr/bin/x86_64-w64-mingw32-gcc-posix -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/x86_64-w64-mingw32-g++-posix
-- Check for working CXX compiler: /usr/bin/x86_64-w64-mingw32-g++-posix -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test HAS_CXX_STD
-- Performing Test HAS_CXX_STD - Success
-- compiler/linker accepts version script? TRUE
-- compiler/linker supports symbol versioning? FALSE
-- mingw detected
-- wine detected
-- Win32 detected
-- Performing Test HAS_CFLAG_MAYBE_UNINITIALIZED
-- Performing Test HAS_CFLAG_MAYBE_UNINITIALIZED - Success
-- C flags are  -std=gnu99   -Wstrict-prototypes  -Wno-long-long -Wpedantic -Wno-variadic-macros -Wall -Wextra -Wno-overlength-strings -Wsign-compare -Wfloat-equal -Wformat -Wformat-security -Wshadow -Wcomments -Wtrigraphs -Wundef -Wuninitialized -Winit-self -Wmaybe-uninitialized -Wsign-compare -Wfloat-equal
-- CXX flags are  -std=c++11   -Wno-missing-field-initializers -Woverloaded-virtual  -Wsign-promo  -Wno-long-long -Wpedantic -Wno-variadic-macros -Wall -Wextra -Wno-overlength-strings -Wsign-compare -Wfloat-equal -Wformat -Wformat-security -Wshadow -Wcomments -Wtrigraphs -Wundef -Wuninitialized -Winit-self -Wmaybe-uninitialized
-- You are building Elektra 0.9.7
-- Looking for mkfifo
-- Looking for mkfifo - not found
-- Looking for fork
-- Looking for fork - not found
-- Exclude plugin process because mkfifo does not exist on the target platform, excluding pluginprocess library, pluginprocess library excluded, thus also excluding the process plugin
-- Looking for __GNU_LIBRARY__
-- Looking for __GNU_LIBRARY__ - not found
-- Exclude plugin simpleini because simpleini needs glibc to work, use the mini plugin instead
-- Exclude plugin cache because resolver plugin not found (silent)
-- Include plugin dump
-- gopts: using gopts_win32.h
-- Include plugin gopts
-- Include plugin list
-- Include plugin spec
-- Include plugin sync
-- Include plugin type
-- Include plugin wresolver
-- Looking for fnmatch
-- Looking for fnmatch - not found
Excluding lib-globbing, because fnmatch was not found.
mkfifo does not exist on the target platform, excluding pluginprocess library
-- Exclude Library notification because internalnotification plugin excluded
-- Include binding cpp
-- Exclude binding jna because it can only be built if `BUILD_SHARED` is enabled
-- Exclude binding rust because excluded by category EXPERIMENTAL
-- Exclude binding python because neither swig2/3/4 found. Please install swig 2/3/4 and set -DSWIG_EXECUTABLE=
-- Exclude binding lua because neither swig2/3/4 found. Please install swig 2/3/4 and set -DSWIG_EXECUTABLE=
-- Exclude binding ruby because explicitly excluded
-- Exclude binding gsettings because excluded by category EXPERIMENTAL
-- Exclude binding intercept_fs because excluded by category EXPERIMENTAL
-- Exclude binding intercept_env because intercept_env only works with BUILD_SHARED
-- Exclude binding io_uv because excluded by category EXPERIMENTAL
-- Exclude binding io_ev because excluded by category EXPERIMENTAL
-- Exclude binding io_glib because excluded by category EXPERIMENTAL
-- Include tool kdb
-- Looking for clearenv
-- Looking for clearenv - not found
-- Looking for setenv
-- Looking for setenv - not found
-- Looking for futimens
-- Looking for futimens - not found
-- Looking for hsearch_r
-- Looking for hsearch_r - not found
-- Looking for futimes
-- Looking for futimes - not found
-- Looking for glob
-- Looking for glob - not found
-- Looking for clock_gettime
-- Looking for clock_gettime - found
-- Looking for ctype.h
-- Looking for ctype.h - found
-- Looking for errno.h
-- Looking for errno.h - found
-- Looking for features.h
-- Looking for features.h - not found
-- Looking for locale.h
-- Looking for locale.h - found
-- Looking for stdio.h
-- Looking for stdio.h - found
-- Looking for stdlib.h
-- Looking for stdlib.h - found
-- Looking for string.h
-- Looking for string.h - found
-- Looking for time.h
-- Looking for time.h - found
-- Looking for unistd.h
-- Looking for unistd.h - found
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of int
-- Check size of int - done
-- Check size of long
-- Check size of long - done
-- Check size of size_t
-- Check size of size_t - done
-- Check size of long long
-- Check size of long long - done
-- Check size of long double
-- Check size of long double - done
-- Check size of mode_t
-- Check size of mode_t - done
-- Check size of float
-- Check size of float - done
-- Check size of double
-- Check size of double - done
-- Check size of ((struct timeval*)0)->tv_usec
-- Check size of ((struct timeval*)0)->tv_usec - done
-- Check size of ((struct stat*)0)->st_size
-- Check size of ((struct stat*)0)->st_size - done
-- Check if the system is big endian
-- Searching 16 bit integer
-- Check size of unsigned short
-- Check size of unsigned short - done
-- Using unsigned short
-- Check if the system is big endian - little endian
-- Configuring done
-- Generating done
-- Build files have been written to: ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build

...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build$ make -j10

Scanning dependencies of target elektra-type-objects
Scanning dependencies of target elektra-gopts-objects
Scanning dependencies of target elektra-dump-objects
Scanning dependencies of target elektra-list-objects
Scanning dependencies of target elektra-spec-objects
Scanning dependencies of target generate_version_script
Scanning dependencies of target elektra-wresolver-objects
Scanning dependencies of target kdb_gen_templates_generated
Scanning dependencies of target elektra-sync-objects
Scanning dependencies of target elektra_config_headers
[  1%] Building C object src/plugins/gopts/CMakeFiles/elektra-gopts-objects.dir/gopts.c.obj
[  1%] Generating ../../../../include/gen/templates.hpp
[  1%] Building C object src/plugins/type/CMakeFiles/elektra-type-objects.dir/type.c.obj
[  1%] Building CXX object src/plugins/dump/CMakeFiles/elektra-dump-objects.dir/dump.cpp.obj
[  1%] Building C object src/plugins/spec/CMakeFiles/elektra-spec-objects.dir/spec.c.obj
[  1%] Building C object src/plugins/wresolver/CMakeFiles/elektra-wresolver-objects.dir/wresolver.c.obj
[  2%] Building C object src/plugins/list/CMakeFiles/elektra-list-objects.dir/list.c.obj
[  2%] Creating version script
[  2%] Building C object src/plugins/sync/CMakeFiles/elektra-sync-objects.dir/sync.c.obj
[  2%] Built target elektra_config_headers
[  2%] Built target generate_version_script
Scanning dependencies of target elektra-opts-objects
[  3%] Building C object src/plugins/type/CMakeFiles/elektra-type-objects.dir/types.c.obj
[  3%] Building C object src/libs/opts/CMakeFiles/elektra-opts-objects.dir/opts.c.obj
In file included from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/plugins/type/types.h:17,
                 from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/plugins/type/types.c:10:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/plugins/type/types.c: In function ‘elektraTypeNormalizeEnum’:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build/src/include/kdbtypes.h:83:38: warning: ISO C does not support the ‘I64’ ms_printf length modifier [-Wformat=]
   83 | #define ELEKTRA_UNSIGNED_LONG_LONG_F "%" PRIu64
      |                                      ^~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build/src/include/kdbtypes.h:83:38: note: in definition of macro ‘ELEKTRA_UNSIGNED_LONG_LONG_F’
   83 | #define ELEKTRA_UNSIGNED_LONG_LONG_F "%" PRIu64
      |                                      ^~~
In file included from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/plugins/gopts/gopts.c:31:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/plugins/gopts/gopts_win32.h: In function ‘loadEnvp’:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/plugins/gopts/gopts_win32.h:56:9: warning: unused variable ‘size’ [-Wunused-variable]
   56 |  size_t size = 0;
      |         ^~~~
[  3%] Built target kdb_gen_templates_generated
[  3%] Built target elektra-gopts-objects
Scanning dependencies of target elektra-ease-objects
[  3%] Built target elektra-type-objects
Scanning dependencies of target elektra-plugin-objects
Scanning dependencies of target elektra-meta-objects
[  4%] Building C object src/libs/plugin/CMakeFiles/elektra-plugin-objects.dir/plugin.c.obj
[  5%] Building C object src/libs/ease/CMakeFiles/elektra-ease-objects.dir/array.c.obj
Scanning dependencies of target elektra-utility-objects
[  5%] Building C object src/libs/meta/CMakeFiles/elektra-meta-objects.dir/meta.c.obj
[  5%] Built target elektra-spec-objects
[  5%] Built target elektra-list-objects
[  5%] Building C object src/libs/ease/CMakeFiles/elektra-ease-objects.dir/conversion.c.obj
[  5%] Building C object src/libs/utility/CMakeFiles/elektra-utility-objects.dir/text.c.obj
Scanning dependencies of target elektra-io-objects
[  5%] Building C object src/libs/io/CMakeFiles/elektra-io-objects.dir/io.c.obj
[  5%] Built target elektra-utility-objects
Scanning dependencies of target elektra-invoke-objects
[  5%] Building C object src/libs/invoke/CMakeFiles/elektra-invoke-objects.dir/invoke.c.obj
[  5%] Built target elektra-sync-objects
Scanning dependencies of target elektra-highlevel-objects
[  5%] Building C object src/libs/highlevel/CMakeFiles/elektra-highlevel-objects.dir/elektra.c.obj
[  5%] Built target elektra-wresolver-objects
Scanning dependencies of target elektra-merge-objects
[  5%] Building C object src/libs/merge/CMakeFiles/elektra-merge-objects.dir/kdbmerge.c.obj
In file included from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/include/kdbease.h:5,
                 from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/ease/array.c:12:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/ease/array.c: In function ‘elektraReadArrayNumber’:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build/src/include/kdbtypes.h:81:29: warning: ISO C does not support the ‘I64’ ms_scanf length modifier [-Wformat=]
   81 | #define ELEKTRA_LONG_LONG_F "%" PRIi64
      |                             ^~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build/src/include/kdbtypes.h:81:29: note: in definition of macro ‘ELEKTRA_LONG_LONG_F’
   81 | #define ELEKTRA_LONG_LONG_F "%" PRIi64
      |                             ^~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build/src/include/kdbtypes.h:81:29: warning: ISO C does not support the ‘I64’ ms_scanf length modifier [-Wformat=]
   81 | #define ELEKTRA_LONG_LONG_F "%" PRIi64
      |                             ^~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build/src/include/kdbtypes.h:81:29: note: in definition of macro ‘ELEKTRA_LONG_LONG_F’
   81 | #define ELEKTRA_LONG_LONG_F "%" PRIi64
      |                             ^~~
In file included from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/include/kdbease.h:5,
                 from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/ease/conversion.c:9:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/ease/conversion.c: In function ‘elektraLongLongToString’:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build/src/include/kdbtypes.h:81:29: warning: ISO C does not support the ‘I64’ ms_printf length modifier [-Wformat=]
   81 | #define ELEKTRA_LONG_LONG_F "%" PRIi64
      |                             ^~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build/src/include/kdbtypes.h:81:29: note: in definition of macro ‘ELEKTRA_LONG_LONG_F’
   81 | #define ELEKTRA_LONG_LONG_F "%" PRIi64
      |                             ^~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/ease/conversion.c: In function ‘elektraUnsignedLongLongToString’:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build/src/include/kdbtypes.h:83:38: warning: ISO C does not support the ‘I64’ ms_printf length modifier [-Wformat=]
   83 | #define ELEKTRA_UNSIGNED_LONG_LONG_F "%" PRIu64
      |                                      ^~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build/src/include/kdbtypes.h:83:38: note: in definition of macro ‘ELEKTRA_UNSIGNED_LONG_LONG_F’
   83 | #define ELEKTRA_UNSIGNED_LONG_LONG_F "%" PRIu64
      |                                      ^~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/ease/conversion.c: In function ‘elektraLongDoubleToString’:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/ease/conversion.c:612:29: warning: unknown conversion type character ‘L’ in format [-Wformat=]
  612 |  return elektraFormat ("%.21Lg", value);
      |                             ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/ease/conversion.c:612:24: warning: too many arguments for format [-Wformat-extra-args]
  612 |  return elektraFormat ("%.21Lg", value);
      |                        ^~~~~~~~
[  5%] Building C object src/libs/highlevel/CMakeFiles/elektra-highlevel-objects.dir/elektra_array_value.c.obj
[  5%] Building C object src/libs/ease/CMakeFiles/elektra-ease-objects.dir/functional.c.obj
[  5%] Built target elektra-opts-objects
[  5%] Built target elektra-plugin-objects
Scanning dependencies of target kdb-objects
[  6%] Building C object src/libs/ease/CMakeFiles/elektra-ease-objects.dir/hash.c.obj
[  6%] Built target elektra-invoke-objects
[  6%] Built target elektra-meta-objects
[  6%] Building C object src/libs/ease/CMakeFiles/elektra-ease-objects.dir/keyname.c.obj
[  7%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/ansicolors.cpp.obj
[  7%] Building C object src/libs/ease/CMakeFiles/elektra-ease-objects.dir/reference.c.obj
[  7%] Built target elektra-io-objects
[  8%] Building C object src/libs/ease/CMakeFiles/elektra-ease-objects.dir/sha-256.c.obj
[  8%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/basename.cpp.obj
[  8%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/cache.cpp.obj
[  9%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/cmdline.cpp.obj
[  9%] Built target elektra-merge-objects
[  9%] Building C object src/libs/highlevel/CMakeFiles/elektra-highlevel-objects.dir/elektra_error.c.obj
[  9%] Built target elektra-dump-objects
[ 10%] Building C object src/libs/highlevel/CMakeFiles/elektra-highlevel-objects.dir/elektra_value.c.obj
[ 10%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/cmerge.cpp.obj
[ 10%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/command.cpp.obj
[ 10%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/complete.cpp.obj
[ 11%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/convert.cpp.obj
[ 11%] Built target elektra-ease-objects
[ 11%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/cp.cpp.obj
[ 11%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/dirname.cpp.obj
[ 12%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/editor.cpp.obj
[ 12%] Built target elektra-highlevel-objects
Scanning dependencies of target elektra-static
[ 12%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/backend.c.obj
[ 13%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/contracts.c.obj
[ 13%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/errors.c.obj
[ 13%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/export.cpp.obj
[ 13%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/external.cpp.obj
[ 13%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/global.c.obj
[ 14%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/internal.c.obj
[ 14%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/file.cpp.obj
[ 15%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/find.cpp.obj
[ 15%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/kdb.c.obj
[ 15%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/gen.cpp.obj
Scanning dependencies of target elektra-full
[ 15%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/backend.c.obj
[ 15%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/kdbenum.c.obj
[ 15%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/contracts.c.obj
[ 15%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/errors.c.obj
[ 15%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/key.c.obj
[ 16%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/keyhelpers.c.obj
[ 16%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/gen/highlevel/common.cpp.obj
[ 17%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/global.c.obj
[ 17%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/internal.c.obj
[ 17%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/keymeta.c.obj
[ 17%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/keyname.c.obj
[ 17%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/kdb.c.obj
[ 18%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/kdbenum.c.obj
[ 18%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/key.c.obj
[ 18%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/keyhelpers.c.obj
[ 18%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/keymeta.c.obj
[ 19%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/keyset.c.obj
[ 19%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/keytest.c.obj
[ 20%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/keyname.c.obj
[ 20%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/keyset.c.obj
[ 21%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/gen/highlevel/enums.cpp.obj
[ 21%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/gen/highlevel/highlevel.cpp.obj
[ 21%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/keytest.c.obj
In file included from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/include/elektra/types.h:4,
                 from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/include/elektra/error.h:12,
                 from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/include/elektra.h:12,
                 from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/include/kdbprivate.h:12,
                 from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/elektra/keyset.c:9:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/elektra/keyset.c: In function ‘elektraWriteArrayNumber’:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build/src/include/kdbtypes.h:83:38: warning: ISO C does not support the ‘I64’ ms_printf length modifier [-Wformat=]
   83 | #define ELEKTRA_UNSIGNED_LONG_LONG_F "%" PRIu64
      |                                      ^~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/elektra/keyset.c:1813:65: note: in expansion of macro ‘ELEKTRA_UNSIGNED_LONG_LONG_F’
 1813 |  if (snprintf (&newName[index], ELEKTRA_MAX_ARRAY_SIZE - index, ELEKTRA_UNSIGNED_LONG_LONG_F, newIndex) < 0)
      |                                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 22%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/keyvalue.c.obj
In file included from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/include/elektra/types.h:4,
                 from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/include/elektra/error.h:12,
                 from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/include/elektra.h:12,
                 from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/include/kdbprivate.h:12,
                 from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/elektra/keyset.c:9:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/elektra/keyset.c: In function ‘elektraWriteArrayNumber’:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/build/src/include/kdbtypes.h:83:38: warning: ISO C does not support the ‘I64’ ms_printf length modifier [-Wformat=]
   83 | #define ELEKTRA_UNSIGNED_LONG_LONG_F "%" PRIu64
      |                                      ^~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/libs/elektra/keyset.c:1813:65: note: in expansion of macro ‘ELEKTRA_UNSIGNED_LONG_LONG_F’
 1813 |  if (snprintf (&newName[index], ELEKTRA_MAX_ARRAY_SIZE - index, ELEKTRA_UNSIGNED_LONG_LONG_F, newIndex) < 0)
      |                                                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 22%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/keyvalue.c.obj
[ 22%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/mount.c.obj
[ 22%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/nolog.c.obj
[ 23%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/opmphm.c.obj
[ 23%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/opmphmpredictor.c.obj
[ 23%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/plugin.c.obj
[ 23%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/mount.c.obj
[ 24%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/nolog.c.obj
[ 24%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/gen/highlevel/structs.cpp.obj
[ 24%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/opmphm.c.obj
[ 24%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/opmphmpredictor.c.obj
[ 25%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/gen/template.cpp.obj
[ 25%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/proposal.c.obj
[ 26%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/plugin.c.obj
[ 26%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/proposal.c.obj
[ 27%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/rand.c.obj
[ 27%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/split.c.obj
[ 27%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/rand.c.obj
[ 28%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/split.c.obj
[ 28%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/trie.c.obj
[ 28%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/__/loader/static.c.obj
[ 28%] Building C object src/libs/elektra/CMakeFiles/elektra-static.dir/exported_symbols.c.obj
[ 28%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/get.cpp.obj
[ 28%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/trie.c.obj
[ 29%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/__/loader/static.c.obj
[ 30%] Linking CXX static library ../../../lib/libelektra-static.a
[ 30%] Built target elektra-static
[ 30%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/globalmount.cpp.obj
[ 30%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/globalumount.cpp.obj
[ 30%] Building C object src/libs/elektra/CMakeFiles/elektra-full.dir/exported_symbols.c.obj
[ 30%] Linking CXX shared library ../../../bin/libelektra-full.dll
Scanning dependencies of target elektratools-static
[ 31%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/backend.cpp.obj
[ 32%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/import.cpp.obj
[ 32%] Built target elektra-full
[ 32%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/backendbuilder.cpp.obj
[ 32%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/backendparser.cpp.obj
[ 32%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/listcommands.cpp.obj
[ 32%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/ls.cpp.obj
Scanning dependencies of target elektratools-full
[ 33%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/backend.cpp.obj
[ 34%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/main.cpp.obj
[ 34%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/merge.cpp.obj
[ 34%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/mergehelper.cpp.obj
[ 35%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/metaget.cpp.obj
[ 35%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/metals.cpp.obj
[ 35%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/metaremove.cpp.obj
[ 35%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/backends.cpp.obj
[ 35%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/metaset.cpp.obj
[ 36%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/mount.cpp.obj
[ 37%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/helper/comparison.cpp.obj
[ 37%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/backendbuilder.cpp.obj
[ 37%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/mountbase.cpp.obj
Scanning dependencies of target cpp_cascading
[ 38%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_cascading.dir/cpp_cascading.cpp.obj
[ 38%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/backendparser.cpp.obj
[ 38%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/helper/keyhelper.cpp.obj
Scanning dependencies of target cpp_example_dup
[ 38%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_example_dup.dir/cpp_example_dup.cpp.obj
Scanning dependencies of target cpp_example_get
[ 38%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_example_get.dir/cpp_example_get.cpp.obj
[ 38%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/automergeconfiguration.cpp.obj
[ 39%] Linking CXX executable ../../../../bin/cpp_example_dup.exe
[ 39%] Built target cpp_example_dup
[ 39%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/mv.cpp.obj
Scanning dependencies of target cpp_example_set
[ 40%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_example_set.dir/cpp_example_set.cpp.obj
[ 40%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/backends.cpp.obj
[ 41%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/nameparthelper.cpp.obj
[ 41%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/namespace.cpp.obj
[ 41%] Linking CXX executable ../../../../bin/cpp_cascading.exe
[ 42%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/automergestrategy.cpp.obj
[ 42%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/plugincheck.cpp.obj
[ 42%] Built target cpp_cascading
[ 42%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/plugininfo.cpp.obj
[ 43%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/pluginlist.cpp.obj
[ 43%] Linking CXX executable ../../../../bin/cpp_example_get.exe
[ 44%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/helper/comparison.cpp.obj
[ 44%] Built target cpp_example_get
Scanning dependencies of target cpp_example_userio
[ 44%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_example_userio.dir/cpp_example_userio.cpp.obj
[ 44%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/importmergeconfiguration.cpp.obj
[ 44%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/helper/keyhelper.cpp.obj
[ 44%] Linking CXX executable ../../../../bin/cpp_example_set.exe
[ 44%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/remount.cpp.obj
[ 44%] Built target cpp_example_set
Scanning dependencies of target cpp_example_hello
[ 44%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/interactivemergestrategy.cpp.obj
[ 45%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_example_hello.dir/cpp_example_hello.cpp.obj
[ 45%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/rm.cpp.obj
[ 46%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/set.cpp.obj
[ 46%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/sget.cpp.obj
[ 46%] Linking CXX executable ../../../../bin/cpp_example_hello.exe
Scanning dependencies of target cpp_example_hierarchy
[ 46%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_example_hierarchy.dir/cpp_example_hierarchy.cpp.obj
[ 46%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/automergeconfiguration.cpp.obj
[ 46%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/mergeconflictstrategy.cpp.obj
[ 46%] Built target cpp_example_hello
[ 47%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/automergestrategy.cpp.obj
[ 47%] Linking CXX executable ../../../../bin/cpp_example_userio.exe
[ 47%] Built target cpp_example_userio
[ 47%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/shell.cpp.obj
[ 48%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/showmeta.cpp.obj
[ 48%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/importmergeconfiguration.cpp.obj
[ 49%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/mergeresult.cpp.obj
[ 49%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/interactivemergestrategy.cpp.obj
[ 49%] Linking CXX executable ../../../../bin/cpp_example_hierarchy.exe
[ 49%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/mergeconflictstrategy.cpp.obj
[ 49%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/mergingkdb.cpp.obj
[ 49%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/specmount.cpp.obj
[ 49%] Built target cpp_example_hierarchy
[ 49%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/metamergestrategy.cpp.obj
[ 50%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/newkeystrategy.cpp.obj
[ 50%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/onesidemergeconfiguration.cpp.obj
[ 50%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/test.cpp.obj
[ 51%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/mergeresult.cpp.obj
[ 51%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/mergingkdb.cpp.obj
[ 51%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/onesidestrategy.cpp.obj
[ 51%] Building CXX object src/tools/kdb/CMakeFiles/kdb-objects.dir/umount.cpp.obj
[ 51%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/metamergestrategy.cpp.obj
[ 52%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/onesidevaluestrategy.cpp.obj
[ 52%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/overwritemergeconfiguration.cpp.obj
[ 53%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/newkeystrategy.cpp.obj
[ 53%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/merging/threewaymerge.cpp.obj
[ 53%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/modules.cpp.obj
[ 53%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/onesidemergeconfiguration.cpp.obj
[ 54%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/plugin.cpp.obj
[ 54%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/plugindatabase.cpp.obj
[ 54%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/onesidestrategy.cpp.obj
[ 54%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/plugins.cpp.obj
[ 55%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/onesidevaluestrategy.cpp.obj
[ 56%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/pluginspec.cpp.obj
[ 56%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/overwritemergeconfiguration.cpp.obj
[ 56%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/merging/threewaymerge.cpp.obj
[ 56%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-static.dir/specreader.cpp.obj
[ 56%] Built target kdb-objects
Scanning dependencies of target cpp_example_iter
[ 56%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_example_iter.dir/cpp_example_iter.cpp.obj
[ 56%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/modules.cpp.obj
[ 57%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/plugin.cpp.obj
[ 57%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/plugindatabase.cpp.obj
Scanning dependencies of target cpp_example_io
[ 58%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_example_io.dir/cpp_example_io.cpp.obj
Scanning dependencies of target cpp_example_iter_name
[ 58%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_example_iter_name.dir/cpp_example_iter_name.cpp.obj
Scanning dependencies of target cpp_example_ks
[ 59%] Linking CXX executable ../../../../bin/cpp_example_iter.exe
[ 59%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_example_ks.dir/cpp_example_ks.cpp.obj
[ 59%] Built target cpp_example_iter
Scanning dependencies of target cpp_example_ordering
[ 59%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_example_ordering.dir/cpp_example_ordering.cpp.obj
[ 59%] Linking CXX executable ../../../../bin/cpp_example_iter_name.exe
Scanning dependencies of target cpp_example_userexception
[ 59%] Building CXX object src/bindings/cpp/examples/CMakeFiles/cpp_example_userexception.dir/cpp_example_userexception.cpp.obj
[ 59%] Built target cpp_example_iter_name
Scanning dependencies of target benchmark_thread
[ 60%] Building CXX object src/bindings/cpp/benchmarks/CMakeFiles/benchmark_thread.dir/benchmark_thread.cpp.obj
Scanning dependencies of target benchmark_sync
[ 60%] Building CXX object src/bindings/cpp/benchmarks/CMakeFiles/benchmark_sync.dir/benchmark_sync.cpp.obj
[ 61%] Linking CXX executable ../../../../bin/cpp_example_ks.exe
In file included from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/bindings/cpp/benchmarks/benchmark_thread.cpp:1200:
/usr/share/mingw-w64/include/winsock2.h:15:2: warning: #warning Please include winsock2.h before windows.h [-Wcpp]
   15 | #warning Please include winsock2.h before windows.h
      |  ^~~~~~~
Scanning dependencies of target set_key
[ 62%] Building C object examples/CMakeFiles/set_key.dir/set_key.c.obj
[ 62%] Built target cpp_example_ks
Scanning dependencies of target keyBasename
[ 62%] Building C object examples/CMakeFiles/keyBasename.dir/keyBasename.c.obj
[ 62%] Linking CXX executable ../../../../bin/cpp_example_ordering.exe
In file included from ...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/src/bindings/cpp/benchmarks/benchmark_sync.cpp:234:
/usr/share/mingw-w64/include/winsock2.h:15:2: warning: #warning Please include winsock2.h before windows.h [-Wcpp]
   15 | #warning Please include winsock2.h before windows.h
      |  ^~~~~~~
[ 62%] Linking CXX executable ../../../../bin/cpp_example_io.exe
[ 62%] Built target cpp_example_ordering
Scanning dependencies of target kdbopen
[ 63%] Linking CXX executable ../../../../bin/cpp_example_userexception.exe
[ 63%] Building C object examples/CMakeFiles/kdbopen.dir/kdbopen.c.obj
[ 63%] Linking C executable ../bin/set_key.exe
[ 63%] Built target cpp_example_userexception
[ 63%] Built target set_key
Scanning dependencies of target kdbget_error
[ 63%] Built target cpp_example_io
[ 64%] Linking C executable ../bin/keyBasename.exe
[ 64%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/plugins.cpp.obj
Scanning dependencies of target reference
[ 64%] Building C object examples/CMakeFiles/kdbget_error.dir/kdbget_error.c.obj
[ 64%] Building C object examples/CMakeFiles/reference.dir/reference.c.obj
[ 64%] Built target keyBasename
Scanning dependencies of target kdbget
[ 64%] Linking CXX static library ../../../../lib/libelektratools-static.a
[ 64%] Building C object examples/CMakeFiles/kdbget.dir/kdbget.c.obj
[ 64%] Built target elektratools-static
Scanning dependencies of target hello
[ 64%] Linking C executable ../bin/kdbopen.exe
[ 65%] Building C object examples/CMakeFiles/hello.dir/hello.c.obj
[ 65%] Built target kdbopen
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c: In function ‘main’:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:32:24: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   32 |  printf ("key has ref %zd\n", keyGetRef (k));
      |                        ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:32:10: warning: too many arguments for format [-Wformat-extra-args]
   32 |  printf ("key has ref %zd\n", keyGetRef (k));
      |          ^~~~~~~~~~~~~~~~~~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:32:24: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   32 |  printf ("key has ref %zd\n", keyGetRef (k));
      |                        ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:32:10: warning: too many arguments for format [-Wformat-extra-args]
   32 |  printf ("key has ref %zd\n", keyGetRef (k));
      |          ^~~~~~~~~~~~~~~~~~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:39:24: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   39 |  printf ("key has ref %zd\n", keyGetRef (k));
      |                        ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:39:10: warning: too many arguments for format [-Wformat-extra-args]
   39 |  printf ("key has ref %zd\n", keyGetRef (k));
      |          ^~~~~~~~~~~~~~~~~~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:39:24: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   39 |  printf ("key has ref %zd\n", keyGetRef (k));
      |                        ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:39:10: warning: too many arguments for format [-Wformat-extra-args]
   39 |  printf ("key has ref %zd\n", keyGetRef (k));
      |          ^~~~~~~~~~~~~~~~~~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:47:24: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   47 |  printf ("key has ref %zd\n", keyGetRef (k));
      |                        ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:47:10: warning: too many arguments for format [-Wformat-extra-args]
   47 |  printf ("key has ref %zd\n", keyGetRef (k));
      |          ^~~~~~~~~~~~~~~~~~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:47:24: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   47 |  printf ("key has ref %zd\n", keyGetRef (k));
      |                        ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:47:10: warning: too many arguments for format [-Wformat-extra-args]
   47 |  printf ("key has ref %zd\n", keyGetRef (k));
      |          ^~~~~~~~~~~~~~~~~~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:52:24: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   52 |  printf ("key has ref %zd\n", keyGetRef (k));
      |                        ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:52:10: warning: too many arguments for format [-Wformat-extra-args]
   52 |  printf ("key has ref %zd\n", keyGetRef (k));
      |          ^~~~~~~~~~~~~~~~~~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:52:24: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   52 |  printf ("key has ref %zd\n", keyGetRef (k));
      |                        ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/reference.c:52:10: warning: too many arguments for format [-Wformat-extra-args]
   52 |  printf ("key has ref %zd\n", keyGetRef (k));
      |          ^~~~~~~~~~~~~~~~~~~
Scanning dependencies of target keyNew
[ 65%] Building C object examples/CMakeFiles/keyNew.dir/keyNew.c.obj
[ 65%] Linking C executable ../bin/reference.exe
[ 65%] Linking C executable ../bin/kdbget_error.exe
[ 65%] Built target kdbget_error
[ 66%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/pluginspec.cpp.obj
[ 66%] Built target reference
[ 66%] Building CXX object src/libs/tools/src/CMakeFiles/elektratools-full.dir/specreader.cpp.obj
[ 67%] Linking C executable ../bin/kdbget.exe
[ 67%] Built target kdbget
Scanning dependencies of target iterate
[ 68%] Building C object examples/CMakeFiles/iterate.dir/iterate.c.obj
[ 68%] Linking C executable ../bin/hello.exe
[ 69%] Linking C executable ../bin/keyNew.exe
[ 69%] Built target hello
Scanning dependencies of target functional
[ 70%] Building C object examples/CMakeFiles/functional.dir/functional.c.obj
[ 70%] Built target keyNew
Scanning dependencies of target keyNewExample
[ 70%] Building C object examples/CMakeFiles/keyNewExample.dir/keyNewExample.c.obj
[ 70%] Linking C executable ../bin/iterate.exe
[ 70%] Built target iterate
Scanning dependencies of target keyMetaKeySet
[ 70%] Building C object examples/CMakeFiles/keyMetaKeySet.dir/keyMetaKeySet.c.obj
[ 70%] Linking C executable ../bin/functional.exe
[ 70%] Built target functional
Scanning dependencies of target cascading
[ 70%] Building C object examples/CMakeFiles/cascading.dir/cascading.c.obj
[ 70%] Linking C executable ../bin/keyNewExample.exe
Scanning dependencies of target gopts
[ 70%] Built target keyNewExample
[ 70%] Building C object examples/CMakeFiles/gopts.dir/gopts.c.obj
Scanning dependencies of target keyName
[ 71%] Building C object examples/CMakeFiles/keyName.dir/keyName.c.obj
Scanning dependencies of target kdbintro
[ 71%] Linking C executable ../bin/keyMetaKeySet.exe
[ 72%] Building C object examples/CMakeFiles/kdbintro.dir/kdbintro.c.obj
[ 72%] Built target keyMetaKeySet
Scanning dependencies of target opts
Scanning dependencies of target ksIterate
[ 72%] Building C object examples/CMakeFiles/opts.dir/opts.c.obj
[ 73%] Building C object examples/CMakeFiles/ksIterate.dir/ksIterate.c.obj
[ 73%] Linking C executable ../bin/cascading.exe
[ 73%] Built target cascading
Scanning dependencies of target keyCopy
[ 73%] Building C object examples/CMakeFiles/keyCopy.dir/keyCopy.c.obj
[ 74%] Linking C executable ../bin/gopts.exe
[ 74%] Linking C executable ../bin/keyName.exe
[ 74%] Linking C executable ../bin/kdbintro.exe
[ 74%] Built target gopts
Scanning dependencies of target kdbset
[ 74%] Built target kdbintro
Scanning dependencies of target helloElektra
[ 74%] Built target keyName
[ 75%] Building C object examples/CMakeFiles/kdbset.dir/kdbset.c.obj
[ 75%] Building C object examples/CMakeFiles/helloElektra.dir/helloElektra.c.obj
Scanning dependencies of target keyset
[ 75%] Linking C executable ../bin/opts.exe
[ 76%] Building C object examples/CMakeFiles/keyset.dir/keyset.c.obj
[ 76%] Linking C executable ../bin/ksIterate.exe
[ 76%] Built target opts
Scanning dependencies of target meta
[ 76%] Built target ksIterate
[ 77%] Building C object examples/CMakeFiles/meta.dir/meta.c.obj
Scanning dependencies of target goptsCommands
[ 77%] Building C object examples/CMakeFiles/goptsCommands.dir/goptsCommands.c.obj
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/helloElektra.c: In function ‘main’:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/helloElektra.c:24:39: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   24 |  printf ("Number of key-value pairs: %zd\n", ksGetSize (config));
      |                                       ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/helloElektra.c:24:10: warning: too many arguments for format [-Wformat-extra-args]
   24 |  printf ("Number of key-value pairs: %zd\n", ksGetSize (config));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/helloElektra.c:24:39: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   24 |  printf ("Number of key-value pairs: %zd\n", ksGetSize (config));
      |                                       ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/helloElektra.c:24:10: warning: too many arguments for format [-Wformat-extra-args]
   24 |  printf ("Number of key-value pairs: %zd\n", ksGetSize (config));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/helloElektra.c:29:39: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   29 |  printf ("Number of key-value pairs: %zd\n", ksGetSize (config));
      |                                       ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/helloElektra.c:29:10: warning: too many arguments for format [-Wformat-extra-args]
   29 |  printf ("Number of key-value pairs: %zd\n", ksGetSize (config));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/helloElektra.c:29:39: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   29 |  printf ("Number of key-value pairs: %zd\n", ksGetSize (config));
      |                                       ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/helloElektra.c:29:10: warning: too many arguments for format [-Wformat-extra-args]
   29 |  printf ("Number of key-value pairs: %zd\n", ksGetSize (config));
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 77%] Linking C executable ../bin/helloElektra.exe
[ 77%] Linking C executable ../bin/keyCopy.exe
[ 77%] Built target helloElektra
Scanning dependencies of target ksCut
[ 77%] Building C object examples/CMakeFiles/ksCut.dir/ksCut.c.obj
[ 77%] Linking C executable ../bin/keyset.exe
[ 77%] Built target keyCopy
Scanning dependencies of target ksNewExample
[ 77%] Building C object examples/CMakeFiles/ksNewExample.dir/ksNewExample.c.obj
[ 77%] Built target keyset
Scanning dependencies of target optsCommands
[ 77%] Linking C executable ../bin/kdbset.exe
[ 77%] Building C object examples/CMakeFiles/optsCommands.dir/optsCommands.c.obj
Scanning dependencies of target ksLookupPop
[ 77%] Building C object examples/CMakeFiles/ksLookupPop.dir/ksLookupPop.c.obj
[ 77%] Built target kdbset
Scanning dependencies of target keyMeta
[ 77%] Linking C executable ../bin/goptsCommands.exe
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/meta.c: In function ‘main’:
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/meta.c:28:61: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   28 |  printf ("Metadata %s has the value %s with the value size %zd\n", keyName (meta), (const char *) keyValue (meta),
      |                                                             ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/meta.c:28:10: warning: too many arguments for format [-Wformat-extra-args]
   28 |  printf ("Metadata %s has the value %s with the value size %zd\n", keyName (meta), (const char *) keyValue (meta),
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/meta.c:28:61: warning: unknown conversion type character ‘z’ in format [-Wformat=]
   28 |  printf ("Metadata %s has the value %s with the value size %zd\n", keyName (meta), (const char *) keyValue (meta),
      |                                                             ^
...PATHTRUNCATEDFORGITHUB.../Repositories/libelektra/examples/meta.c:28:10: warning: too many arguments for format [-Wformat-extra-args]
   28 |  printf ("Metadata %s has the value %s with the value size %zd\n", keyName (meta), (const char *) keyValue (meta),
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 78%] Building C object examples/CMakeFiles/keyMeta.dir/keyMeta.c.obj
[ 78%] Linking C executable ../bin/meta.exe
[ 78%] Built target meta
[ 78%] Built target goptsCommands
Scanning dependencies of target namespace
Scanning dependencies of target ksNew
[ 78%] Building C object examples/CMakeFiles/namespace.dir/namespace.c.obj
[ 78%] Building C object examples/CMakeFiles/ksNew.dir/ksNew.c.obj
[ 78%] Linking C executable ../bin/ksCut.exe
[ 78%] Linking CXX shared library ../../../../bin/libelektratools-full.dll
[ 79%] Linking C executable ../bin/ksNewExample.exe
[ 79%] Built target ksCut
Scanning dependencies of target kdb-static
[ 80%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/ansicolors.cpp.obj
[ 80%] Built target ksNewExample
[ 80%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/basename.cpp.obj
[ 81%] Linking C executable ../bin/optsCommands.exe
[ 82%] Linking C executable ../bin/ksLookupPop.exe
[ 82%] Built target optsCommands
[ 82%] Linking C executable ../bin/keyMeta.exe
[ 82%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/cache.cpp.obj
[ 82%] Built target ksLookupPop
[ 83%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/cmdline.cpp.obj
[ 83%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/cmerge.cpp.obj
[ 83%] Built target keyMeta
[ 83%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/command.cpp.obj
[ 83%] Linking C executable ../bin/ksNew.exe
[ 84%] Linking C executable ../bin/namespace.exe
[ 84%] Built target ksNew
[ 84%] Built target namespace
[ 85%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/complete.cpp.obj
[ 85%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/convert.cpp.obj
[ 85%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/cp.cpp.obj
[ 85%] Built target elektratools-full
Scanning dependencies of target merging
[ 85%] Building CXX object src/libs/tools/examples/CMakeFiles/merging.dir/merging.cpp.obj
[ 85%] Linking CXX executable ../../../../bin/benchmark_sync.exe
[ 85%] Built target benchmark_sync
Scanning dependencies of target backend
[ 85%] Building CXX object src/libs/tools/examples/CMakeFiles/backend.dir/backend.cpp.obj
[ 85%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/dirname.cpp.obj
[ 86%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/editor.cpp.obj
[ 86%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/export.cpp.obj
[ 86%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/external.cpp.obj
[ 87%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/file.cpp.obj
[ 87%] Linking CXX executable ../../../../bin/merging.exe
[ 87%] Linking CXX executable ../../../../bin/benchmark_thread.exe
[ 87%] Built target merging
[ 87%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/find.cpp.obj
[ 87%] Linking CXX executable ../../../../bin/backend.exe
[ 87%] Built target backend
[ 87%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/gen.cpp.obj
[ 87%] Built target benchmark_thread
Scanning dependencies of target benchmark_plugins
[ 87%] Building CXX object src/libs/tools/benchmarks/CMakeFiles/benchmark_plugins.dir/benchmark_plugins.cpp.obj
[ 87%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/gen/highlevel/common.cpp.obj
Scanning dependencies of target kdb-full
[ 88%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/gen/highlevel/enums.cpp.obj
[ 88%] Linking CXX executable ../../../bin/kdb-full.exe
[ 88%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/gen/highlevel/highlevel.cpp.obj
[ 88%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/gen/highlevel/structs.cpp.obj
[ 89%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/gen/template.cpp.obj
[ 89%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/get.cpp.obj
[ 89%] Built target kdb-full
[ 89%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/globalmount.cpp.obj
[ 90%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/globalumount.cpp.obj
[ 91%] Linking CXX executable ../../../../bin/benchmark_plugins.exe
[ 91%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/import.cpp.obj
[ 91%] Built target benchmark_plugins
[ 91%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/listcommands.cpp.obj
[ 91%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/ls.cpp.obj
[ 92%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/main.cpp.obj
[ 92%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/merge.cpp.obj
[ 92%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/mergehelper.cpp.obj
[ 93%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/metaget.cpp.obj
[ 93%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/metals.cpp.obj
[ 93%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/metaremove.cpp.obj
[ 93%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/metaset.cpp.obj
[ 94%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/mount.cpp.obj
[ 94%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/mountbase.cpp.obj
[ 94%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/mv.cpp.obj
[ 95%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/nameparthelper.cpp.obj
[ 95%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/namespace.cpp.obj
[ 95%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/plugincheck.cpp.obj
[ 95%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/pluginlist.cpp.obj
[ 96%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/plugininfo.cpp.obj
[ 96%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/remount.cpp.obj
[ 96%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/rm.cpp.obj
[ 97%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/set.cpp.obj
[ 97%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/sget.cpp.obj
[ 97%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/shell.cpp.obj
[ 98%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/showmeta.cpp.obj
[ 98%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/specmount.cpp.obj
[ 98%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/test.cpp.obj
[100%] Building CXX object src/tools/kdb/CMakeFiles/kdb-static.dir/umount.cpp.obj
[100%] Linking CXX executable ../../../bin/kdb-static.exe
[100%] Built target kdb-static
```

**Comment by flo91** [comment]
@markus2330 Maybe another issue for the FLOSS course?

**Comment by markus2330** [comment]
Absolutely, it is very easy to reproduce and at least some of the warnings should be not to difficult to fix. (E.g. defining ELEKTRA_UNSIGNED_LONG_LONG_F different for mingw)
